blueprint/office-365.md

```diff
@@ -1627,3 +1615,53 @@ Add trust senders and domains | Not configured | When users interact with domain
 Spoofing filter settings | Enable antispoofing protection: On | Allows the Agency to filter email from senders who are spoofing domains.
 Enable Unauthenticated Sender Feature | Enable Unauthenticated Sender: On | Displays a notification to users in Outlook when a sender fails authentication checks.
 Actions | If email is sent by someone who's not allowed to spoof your domain: Quarantine the message | Specify an action in an event an unauthorised user spoofs a domain.
+
+## Microsoft Whiteboard
+Microsoft Whiteboard is a hosted whiteboarding tool used for collaboration. It integrates with Office 365 services such as Teams and functions on devices such as the Microsoft Surface hub, Windows and iOS. Acess to use the tool can be controlled globally.
+
+Microsoft leveages diagnostic data to both address issues and develop updates to the Microsoft Whiteboard tool. The level of data provided to Microsoft from clients utilising the Whiteboard application can be controlled. 
+
+Diagnostic data level options are:
```

Comment:
Join these paragraphs, they are related

Typo: leveages

Suggest simplifying:

`Microsoft uses diagnostic data to address issues and continually improve Microsoft Whiteboard...`